frontend/lib/ui/KeyboardKey/Enter.tsx

```diff
@@ -0,0 +1,13 @@
+import { IconCornerDownLeft } from "@kiesraad/icon";
+import { cn } from "@kiesraad/util";
+
+import cls from "./KeyboardKey.module.css";
+
+export function Enter() {
+  return (
+    <div className={cn(cls["keyboard-key"])}>
+      <span>Enter</span>
+      <IconCornerDownLeft />
+    </div>
```

Comment:
We should use the [`<kbd>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/kbd) here.

Also, did you consider using Unicode symbols instead of an icon? (e.g. <kbd>⇧ Shift</kbd> <kbd>Enter ⏎</kbd>)

Reply:
![image](https://github.com/user-attachments/assets/9d5f343d-5cf7-456b-8e57-89dbe0ee469c)
I think having non unicode icons looks a bit better. The one with unicode has the <kbd> element around it.

Reply:
I think we can make the Unicode icons look a bit better still, but let's just stick to the SVG icons instead.

Reply:
Agreed, let's stick to the svg. 
regarding the `<kbd>` element: semantically better. Can we override the font so it's the same as in design?

Reply:
Yes I have just made the changes, please have a look

Reply:
Yeah saw your PR. Looks good to me! 👌